shell/platform/darwin/ios/framework/Source/FlutterEngineGroup.mm

```diff
@@ -5,45 +5,30 @@
 #import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterEngineGroup.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h"
 
-@implementation FlutterEngineGroupOptions
-
-- (void)dealloc {
-  [_entrypoint release];
-  [_libraryURI release];
-  [_initialRoute release];
-  [_entrypointArgs release];
-  [super dealloc];
-}
+FLUTTER_ASSERT_ARC
 
+@implementation FlutterEngineGroupOptions
 @end
 
 @interface FlutterEngineGroup ()
 @property(nonatomic, copy) NSString* name;
-@property(nonatomic, retain) NSMutableArray<NSValue*>* engines;
-@property(nonatomic, retain) FlutterDartProject* project;
+@property(nonatomic, strong) NSMutableArray<NSValue*>* engines;
+@property(nonatomic, strong) FlutterDartProject* project;
+@property(nonatomic, assign) int enginesCreatedCount;
```

Comment:
super nit doesn't matter: Probably should be an NSUInteger and the string format changed to `%lu` but it doesn't really matter given how it's used...

Reply:
Done!